lib/data/monorepo.json

```diff
@@ -648,6 +648,7 @@
     "spock": "/^org\\.spockframework:spock-/",
     "syncfusion-dotnet": "/^Syncfusion\\./",
     "testing-library": "/^@testing-library//",
+    "twig": "/^twig//",
```

Comment:
This seems very generic as it will apply across ecosystems

Reply:
I think here makes most sense: https://github.com/renovatebot/renovate/blob/3169d1a9dd55e2003a25cc24943709b7ad63bc55/lib/data/monorepo.json#L619-L627

Reply:
looks like a good place indeed. adapted, thx!